x-pack/test/apm_api_integration/tests/transactions/transactions_groups_comparison_statistics.ts

```diff
@@ -70,19 +71,19 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
         expect(response.status).to.be(200);
 
-        const transactionsGroupsComparisonStatistics = response.body as TransactionsGroupsComparisonStatistics;
+        const {
+          currentPeriod,
+          previousPeriod,
+        } = response.body as TransactionsGroupsComparisonStatistics;
 
-        expect(Object.keys(transactionsGroupsComparisonStatistics).length).to.be.eql(
-          transactionNames.length
-        );
+        expect(Object.keys(currentPeriod).length).to.be.eql(transactionNames.length);
```

Comment:
I think comparing Object.keys(currentPeriod) to transactionNames is more useful here.